src/pages/Registration/Registration.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import './Registration.scss';
+import { Button, Form } from 'react-bootstrap';
+import axios from "axios";
+import alertify from 'alertifyjs';
+const Registration = () => {
+    const [name, setName] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const onFormSubmitHandler = event => {
+        event.preventDefault();
+        const data = { name, email, password };
+        axios
+            .post("https://social-nodejs-be.herokuapp.com/api/user/signup", data)
```

Comment:
Keep this Base URL in .env file
Base URL: https://social-nodejs-be.herokuapp.com

rest URL should be in a constant file:
/api/user/signup

Reply:
Done.